docs/v4/mini-faq.md

```diff
@@ -0,0 +1,198 @@
+
+**- Ok, but I feel that I want to go with the pure v4, how I start?** There is a quickstart here: https://github.com/axa-group/nlp.js/blob/master/docs/v4/quickstart.md
+
+**- But this is for backend... I want this bundle to work in my browser or mobile using react native** You have the quickstart for browser and react native here: https://github.com/axa-group/nlp.js/blob/master/docs/v4/webandreact.md
+
+**- This thing of intents is hard to understand, I just want to do a Questions and Answers bot** You have a quickstart for simple QnA here: https://github.com/axa-group/nlp.js/blob/master/docs/v4/qna.md
+
+**- How I test my chatbot in Console?** https://github.com/axa-group/nlp.js/blob/master/docs/v4/quickstart.md#adding-your-first-connector
+
+**- How I do a multilanguage chatbot?** Installing the package for the language and adding it as a plugin https://github.com/axa-group/nlp.js/blob/master/docs/v4/quickstart.md#adding-multilanguage
+
+**- Do you have some example of a chatbot running in several languages?** You can remix this project in glitch, you'll see that it only haves one line of source code, but with only one line it creates a backend with API and exposes a react frontend with the bot, and it's multi-language. To see the frontend click on the button "show" and then "next to the code". https://glitch.com/edit/?utm_content=project_nlpjs-multi&utm_source=remix_this&utm_medium=button&utm_campaign=glitchButton#!/remix/nlpjs-multi
+
+**- Using node-nlp package I need to install the languages separately?** No. It use a pacakge @nlpjs/lang-all that mounts all the languages
+
+**- Where I can see the languages and their locales to find the correct package to install?** Here, the one with Native Support https://github.com/axa-group/nlp.js/blob/master/docs/v4/language-support.md
+
+**- But I want to have a web for my chatbot!** Here you have how to easely expose your chatbot with directline API, and how to expose a WebChat: https://github.com/axa-group/nlp.js/blob/master/docs/v4/quickstart.md#adding-api-and-webchat
+
+**- But this does not help on how to orchestrate a chatbot** Well, NLP.js is the set of NLP tools, not the chatbot ones. For orchestrating a chatbot I recommend you to use Microsoft Bot Framework
+
+**- When an intent is triggered I want to get the answer from an API call and I'm not using any chatbot orchestrating SDK** You can with pipelines that react to your intent, you've an example here: https://github.com/axa-group/nlp.js/blob/master/docs/v4/quickstart.md#adding-logic-to-an-intent
+
+**- How I guess the language from an utterance when I have a multi-language bot?** The language is guessed automatically using the most common 3-grams from the language, but also with the 3-grams from the corpus training it, so that way you can use even languages that does not exists, or get a better guessing based on your corpus.
```

Comment:
rephrase to "How nlp guesses the language from an utterance in a multi-language bot?" to be even clear.